.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - 'iojs'
+  - 'stable'
```

Comment:
It's not directly related to promisifying the library but it is related to the tests which had to be rewritten because of the promises.